### ISSUE TITLE
sign in 

### ISSUE THREAD

**Issue by stitch7ssundee** [brillout/clocktab#8]
you should add the ability to sign in so people can save customs to profiles. So if you have multiple devices you don't need to share a bunch long links to have the same customs and ad free screens.

**Comment by brillout** [comment]
I agree. That said it would involve considerable changes in server infrastructure; I'll implement this if many people ask for it.

**Comment by itskhaledmohammad** [comment]
@brillout what if, rather than implementing the authentication and authorization on your server, we can use a hosted nosql db, such as firebase. That way much won't be needed to be changed.

**Comment by brillout** [comment]
True - still a lot of work though. Feel free to hack through the source code, it's open source after all.
